ddtrace/internal/rate_limiter.py

```diff
@@ -18,6 +21,17 @@ class RateLimiter(_RateLimiter):
     def _has_been_configured(self):
         return self.rate_limit != DEFAULT_SAMPLING_RATE_LIMIT
 
+    def is_allowed(self, timestamp_ns: Optional[int] = None) -> bool:
+        if timestamp_ns is not None:
+            deprecate(
+                "The `timestamp_ns` parameter is deprecated and will be removed in a future version."
+                "Ratelimiter will use the current time.",
+                category=DDTraceDeprecationWarning,
+            )
+        # rate limits are tested and mocked in pytest so we need to compute the timestamp here
+        # (or move the unit tests to rust)
+        return self._is_allowed(compat.monotonic_ns())  # type: ignore[attr-defined]
```

Comment:
```suggestion
        return super().is_allowed(compat.monotonic_ns())
```

?

Reply:
the ignore attr-defined is probably because you didn't update the `internal/core/_core.pyi` ?